app/views/categories/_top_links.html.erb

```diff
@@ -2,7 +2,7 @@
 <ul class="category-link-list">
   <% category.links.each do |link| %>
     <li class="category-link-list__item">
-      <a class="category-link-list__item__link" href="<%= link['url'] %>"><%= link['text'] %></a>
+      <a class="category-link-list__item__link" href="<%= link['url'] %>"><%= link['text'] %><%= ',' unless link === category.links.last %></a>
```

Comment:
As far as the comma after the space, I intentionally left that out as the spacing is added more explicitly via CSS margin.

What I was trying to do with the === was an object comparison (to see if they were the same object), as opposed to a value comparison. It seems like that is not really a thing, however == does not do it either. So I have settled on ".equals?" which works. Sound good?